packages/mdc-top-app-bar/index.ts

```diff
@@ -21,39 +21,31 @@
  * THE SOFTWARE.
  */
 
-import MDCTopAppBarAdapter from './adapter';
 import {MDCComponent} from '@material/base/component';
 import {MDCRipple} from '@material/ripple/index';
+import {MDCTopAppBarAdapter} from './adapter';
 import {cssClasses, strings} from './constants';
-import MDCTopAppBarBaseFoundation from './foundation';
-import MDCFixedTopAppBarFoundation from './fixed/foundation';
-import MDCShortTopAppBarFoundation from './short/foundation';
-import MDCTopAppBarFoundation from './standard/foundation';
+import {MDCFixedTopAppBarFoundation} from './fixed/foundation';
+import {MDCTopAppBarBaseFoundation} from './foundation';
+import {MDCShortTopAppBarFoundation} from './short/foundation';
+import {MDCTopAppBarFoundation} from './standard/foundation';
 
-/**
- * @extends {MDCComponent<!MDCTopAppBarBaseFoundation>}
- * @final
- */
-class MDCTopAppBar extends MDCComponent {
-  /**
-   * @param {...?} args
-   */
-  constructor(...args) {
-    super(...args);
-    /** @private {?Element} */
-    this.navIcon_;
-    /** @type {?Array<MDCRipple>} */
-    this.iconRipples_;
-    /** @type {Object} */
-    this.scrollTarget_;
+export type MDCRippleFactory = (el: Element) => MDCRipple;
```

Comment:
I guess this will end up removed in #4407 after this is merged?

Reply:
Correct 😀  Thanks for the reminder!